packages/cli/test/unit/commands/env/pull.test.ts

```diff
@@ -481,7 +481,71 @@ describe('env pull', () => {
     expect(rawDevEnv.toString().includes('VERCEL_ANALYTICS_ID')).toBeFalsy();
   });
 
-  describe.todo('[filename]');
-  describe.todo('--yes');
-  describe.todo('--git-branch');
+  describe('[filename]', () => {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** use existing tests

We can add these metrics assertions to existing tests. I did this in my attempt here: https://github.com/vercel/vercel/pull/12369/files#diff-f323de450c6e0956c9afe44145acae8e46e44b58d7017ebef9abdef006cffd6dR39

I'm not against explicit tests for them, though.‏

Reply:
One thing the single übertest doesn't protect us from us is accidentally passing the value to telemetry even when it wasn't passed to the CLI.